content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -199,10 +199,36 @@ const result = await prisma.queryRaw(
 )
 ```
 
+The following example uses a PostgreSQL query:
+
+```ts
+const userName = 'Sarah'
+const email = 'alice@prisma.io'
+const result = await prisma.queryRaw(
+  'SELECT * FROM User WHERE (name = $1 OR email = $2)', // PostgreSQL variables
+  userName,
+  email
+)
+```
+
 As with tagged templates, Prisma Client escapes all variables.
 
 > **Note**: You cannot pass a table or column name as a variable into a parameterized query. For example, you cannot `SELECT ?` and pass in `*` or `id, name` based on some condition.
 
+#### Parameterized PostgreSQL `ILIKE` query
+
+When you use `ILIKE`, the `%` wildcard character(s) should be included in the variable itself, not the query string:
```

Comment:
Hm, is "query string" the correct term here? (I only think of URL query strings here, so may be biased and just not know the correct term)

Reply:
Hmm.. you're right, I'll change it to query (`string`) or something like that.